.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -4,14 +4,23 @@ about: Create a report to help us improve
 labels: t:bug
 ---
 
-**Environment**
+## Bug Report
 
-- **CKB version**: output of `ckb --version`
-- **Chain**: dev / staging / testnet / mainnet ...
-- **Operating system**: Linux / macOS / Windows
+<!-- Thanks for your bug report! We appreciate your time to fill following sections to help us locate the root cause. -->
+
+### Environment
+
+- **CKB version**: <!-- output of `ckb --version` -->
```

Comment:
Preview of this line is:

- **CKB version**: <!-- output of `ckb --version` -->

The `<!-- -->` doesn't work when there are "--" in middle of it.